### ISSUE TITLE
[BUG] Using Napalm Altfire causes crash when shot in a room in E3M1 of Blood.

### ISSUE THREAD

**Issue by De-M-oN** [ZDoom/Raze#1041]
### Raze version

1.8.1

### Which game are you running with Raze?

Blood

### What Operating System are you using?

Windows 10

### Relevant hardware info

intel 9900k, RTX 2080 Ti, 64 GB RAM DDR4 3200

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

As you can see on this Youtube Video:

https://youtu.be/3VV-ZyOQyoY

When I shoot a napalm rocket alt fire into this room of E3M1, raze loves to crash out.
Not sure if its the butcher causing it or whatever it is. But I've seen regular death and gibbed death happening.

### Steps to reproduce the behaviour.

Do the same as on my youtube video. Shoot altfire napalm into this room. Repeat until raze crashes. It seems to be a specific angle where you fire. Maybe run around the room and try again. Sometimes it works several times, sometimes its very frequently. It really seems to depend what happens in that room - so how you fire the rocket in it.

### Your configuration

```ini
Would know of nothing relevant to this.
I have weapon interpolation on. Whatever it does.
Anything else I see no relation to config data with this bug
```

### Provide a Log

[raze.log](https://github.com/ZDoom/Raze/files/13721170/raze.log)
[CrashReport.zip](https://github.com/ZDoom/Raze/files/13721177/CrashReport.zip)

**Comment by JarDrh99** [comment]
I tested on the most recent master, i ran around the level firing the secondary napalm and i got the crash several times.
It happened in several indoor rooms including the one you pointed.

**Comment by eqagunn** [comment]
Not sure if related, but I noticed that if you noclip out-of-bounds and fire a primary (or secondary) fire with napalm launcher it will always crash Raze.

Perhaps what happens with secondary fire is that one of the flames jump out-of-bounds, explodes there and crashes Raze the same way.

**Comment by coelckers** [comment]
I can't get it to crash so right now I cannot say for sure what happens.

**Comment by FitzRoyX** [comment]
> I can't get it to crash so right now I cannot say for sure what happens.

I can get it to crash in the first level here. Just load the save and repeatedly alt fire as fast as you can where the crosshair is already pointed.

[save0004.zip](https://github.com/ZDoom/Raze/files/13731235/save0004.zip)

**Comment by coelckers** [comment]
fixed in 485ad27a5adca08348b4a1ee9f460da197967979

Thanks for the save, this was extremely hard to trigger, even with that I often needed to reload several times before the crash happened.

**Comment by De-M-oN** [comment]
Interesting. In the E3M1 room I got it quite often :D But nice that you could fix it :)
